test/test_google/test_google_slides.py

```diff
@@ -0,0 +1,71 @@
+import unittest
+import os
+from parsons import GoogleSlides
+
+# Test Slides: https://docs.google.com/presentation/d/19I-kicyaJV53KoPNwt77KJL10fHzWFdZ_c2mW4XJaxc
+
+
+@unittest.skipIf(
+    not os.environ.get("LIVE_TEST"), "Skipping because not running live test"
+)
```

Comment:
You've set this up so that all of your tests will be skipped unless the environmental variable is set to `LIVE_TEST`. It's great to have live tests, but we also want tests using Mocks that can be run as part of the regular automated testing pipeline. Can you also write some mock versions?

Reply:
Also, for the live tests, will everyone have access to the test slides you've created? If not, instead of hard coding that variable into the tests, we likely want to create another environmental variable people can use.

Reply:
> Also, for the live tests, will everyone have access to the test slides you've created? If not, instead of hard coding that variable into the tests, we likely want to create another environmental variable people can use.

YES the Test Slides are public to everyone on the internet (they contain truly nothing of value). I think given the nature of the tests - in terms of replacing certain text and images, we need to use a standard default.